express/scripts/scripts.js

```diff
@@ -128,6 +129,7 @@ const miloLibs = setLibs(LIBS);
   document.head.append(footerMeta);
 
   listenMiloEvents();
+  replaceContent( document.getElementsByTagName('main')[0], miloLibs)
```

Comment:
this is not great, because you're making every single one of our pages run this script. Create pages don't need this. Quick action pages don't need this. We need to be really careful to only dynamically import content when we actually need it.

Reply:
I implemented our approach from express, so now it checks if a block is spreadsheet powered before running the script. I also used a smaller version @qiyundai gave me.